cmpserver/plugin/plugin.go

```diff
@@ -55,17 +66,26 @@ func environ(envVars []*apiclient.EnvEntry) []string {
 
 // GenerateManifest runs generate command from plugin config file and returns generated manifest files
 func (s *Service) GenerateManifest(ctx context.Context, q *apiclient.ManifestRequest) (*apiclient.ManifestResponse, error) {
+	bufferedCtx, cancel := buffered_context.WithEarlierDeadline(ctx, 100*time.Millisecond)
```

Comment:
Is 100ms reasonable? Should it be relocated to somewhere with other constants? Should it be configurable?